Makefile

```diff
@@ -50,7 +50,7 @@ run-local:
 	SAAS_BASE_URL="https://meshery.layer5.io" \
 	PORT=9081 \
 	DEBUG=true \
-	ADAPTER_URLS="mesherylocal.layer5.io:10000 mesherylocal.layer5.io:10001" \
+	ADAPTER_URLS="mesherylocal.layer5.io:10000 mesherylocal.layer5.io:10001 mesherylocal.layer5.io:10004" \
```

Comment:
@girishranganathan should we bring in Octarine here as well?

Reply:
yes it was already done in the master. Now I have syncd with master and it should be all fine.

Reply:
Thank you. Good to merge?